charts/dbildungscloud-iam-keycloak/values.yaml

```diff
@@ -2,6 +2,5 @@ replicaCount: 1
 keycloakHostname: "keycloak.dev.spsh.dbildungsplattform.de"
 keycloakPublic: false
 image:
-  repository: ghcr.io/dbildungsplattform/dbildungs-iam
-  tag: {{ .Chart.AppVersion }}
+  repository: ghcr.io/dbildungsplattform/dbildungs-iam:{{ .Chart.AppVersion }}
```

Comment:
As we discussed, we want to add a tag to the image as shown here:
**image:
  repository: ghcr.io/dbildungsplattform/dbildungs-iam
  tag: {{ .Chart.AppVersion }}**
However, this implementation results in the following error:

**Error: cannot load values.yaml: error converting YAML to JSON: yaml: invalid map key: map[interface {}]interface {}{".Chart.AppVersion":interface {}(nil)}**
The error indicates that the syntax used for the tag, {{ .Chart.AppVersion }}, is incorrect in YAML. The correct solution is to modify it as follows:

**repository: ghcr.io/dbildungsplattform/dbildungs-iam:{{ .Chart.AppVersion }}**